climada_petals/entity/impact_funcs/test/test_fl.py

```diff
@@ -35,116 +35,114 @@ def test_flood_imp_func_set(self):
 
     def test_region_Africa(self):
```

Comment:
Would it not be simpler to have tests that loop through the allowed parameters and all methods? This class just stores data, so this is what should be tested. Also, a test for when wrong parameters are used would be good. This makes the test more complete, and also shorter.

Reply:
That would be a good idea. The code base is constantly improving, and thus copying older, less mature code is suboptimal going forward.